### PR TITLE
Fix: #1742 Fixed FAQS forgotten state

### LINKED ISSUE #1742: Help page FAQs state is forgotten on screen orientation
**Summary:**
Help page FAQs state is forgotten on screen orientation

**Steps to reproduce:**
1. Open Help Section
2. Open any of the given FAQ
3. Rotate the screen
4. Try opening another FAQ
5. Observe

**Expected behaviour:**
The previous FAQ should close and the new one should open

**Observed behaviour:**
The previous FAQ is not closed and the new one opens

**Device and Android version:**
OnePlus 6
Android 10

**Screenshots:**

https://user-images.githubusercontent.com/50913976/108683628-def18580-7517-11eb-966c-a0ef388f2576.mp4

### PR DESCRIPTION
**Pull request by SekhGulamMainuddin**
Fixes #1742 Fixed Help Page FAQs forgotten state

[Fixed FAQS forgotten state.webm](https://github.com/openMF/mifos-mobile/assets/73953395/674f5d78-fa6f-4d54-9ae7-3f9e99dbaf4e)

### CODE REVIEW

**Comment by SekhGulamMainuddin** [comment]
@PratyushSingh07 can you please review this PR

File: app/src/main/java/org/mifos/mobile/ui/activities/HelpActivity.kt [hunk 1]
```diff
@@ -1,15 +1,19 @@
 package org.mifos.mobile.ui.activities
 
 import android.os.Bundle
+import androidx.activity.viewModels
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/ui/activities/HelpActivity.kt]: where are we using this import in our activity ?

File: app/src/main/java/org/mifos/mobile/ui/activities/HelpActivity.kt [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+import dagger.hilt.android.AndroidEntryPoint
 import org.mifos.mobile.R
 import org.mifos.mobile.databinding.ActivityContainerBinding
 import org.mifos.mobile.ui.activities.base.BaseActivity
 import org.mifos.mobile.ui.fragments.HelpFragment
+import org.mifos.mobile.viewModels.HelpViewModel
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/ui/activities/HelpActivity.kt]: do we need this in our activity ?

File: app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt [hunk 3]
```diff
@@ -70,6 +67,10 @@ class HelpFragment : BaseFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        faqAdapter = FAQAdapter {
+            viewModel.alreadySelectedPosition = it
+            Log.d("currentSelectedPosition", "onViewCreated: $it")
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt]: remove this

File: app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt [hunk 4]
```diff
@@ -196,6 +208,7 @@ class HelpFragment : BaseFragment() {
 
     override fun onDestroyView() {
         _binding = null
+        Log.d("currentSelectedPosition", "onDestroyView: ${viewModel.alreadySelectedPosition}")
```
Comment (PratyushSingh07) [inline app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt]: remove this as well

**Comment by SekhGulamMainuddin** [comment]
@PratyushSingh07 I have made the changes can you please review this PR.

**Comment by SekhGulamMainuddin** [comment]
Hi @therajanmaurya can you please review this PR.

**Comment by PratyushSingh07** [comment]
solve the merge conflicts @SekhGulamMainuddin

**Comment by SekhGulamMainuddin** [comment]
@PratyushSingh07 solved the merge conflicts

**Comment by PratyushSingh07** [comment]
LGTM , we can merge this @therajanmaurya

**Review by PratyushSingh07 — APPROVED** [review]

**Pull request merged**